docs/sections/workflows/include/code/workchain/addcalcjobworkchain.py

```diff
@@ -0,0 +1,39 @@
+from aiida.orm import Int, Code
+from aiida.engine import WorkChain, calcfunction, ToContext
+from aiida.plugins.factories import CalculationFactory
+
+ArithmeticAddCalculation = CalculationFactory("arithmetic.add")
+
+
+class AddCalcjobWorkChain(WorkChain):
```

Comment:
Usually, I would keep the work chain name associated with its functionality rather than the means. But that is not important. So, it is fine like that too.